native/response/asset.go

```diff
@@ -0,0 +1,97 @@
+package response
+
+import "github.com/mxmCherry/openrtb"
```

Comment:
I have a feeling that it's not the best idea.

Better don't reuse simple stuff like `RawJSON` here. Just copy that file/code - it's fine to do this, since it should really be thrown out in favor of `json.RawMessage` (when Go 1.10 is out). Probs, you can even add some deprecation warning to it's godoc-comment.

I think, `native/*` stuff may need to be imported in `openrtb` package itself.

Like:

```go
type openrtb.Native struct {
  Request NativeWrapper
}

type NativeWrapper struct {
  Native openrtb/native/request.Request
}

func (NativeWrapper) UnmarshalJSON() error {
  // unmarshal stringified JSON or raw JSON object
}
```

Better don't bother with `NativeWrapper` for now, but make it possible to be added later (that may happen before `RawJSON` goes away).